src/utils/validations/common.ts

```diff
@@ -10,7 +10,7 @@ const validations: Validations = {
     if (value.length > 30) {
       return 'common.errorMessages.nameLength'
     }
-    if (!RegExp(/^(?! )[a-zа-яєії ]+(?<! )$/i).test(value)) {
+    if (!RegExp(/^[a-zа-яєії]+$/i).test(value)) {
```

Comment:
Is there a need to use RegExp constructor or we can omit it?

Reply:
You are right. I guess it would be even better to make a separate object with validation patterns.